src/openassetio-python/tests/package/test_entityreference.py

```diff
@@ -42,9 +42,9 @@ def test_returns_constructor_supplied_string(self):
 
 
 class Test_EntityReferenve_repr:
-    def test_returns_angle_bracket_string_with_class_and_value(self):
+    def test_returns_bracket_string_with_class_and_value(self):
```

Comment:
[question] Minor, just curious why keep this test when others were removed?

Reply:
No need to be curious, just not thinking as usual.
https://github.com/OpenAssetIO/OpenAssetIO/pull/1329/commits/3c871896e1e0176d12448b798505fde2c2e3d321